src/browser/modules/Stream/PlayFrame.jsx

```diff
@@ -152,6 +151,22 @@ function generateContent(stackFrame, bus, onSlide, shouldUseSlidePointer) {
 
   // Found a remote guide
   if (stackFrame.result) {
+    if (['md', 'mdx'].includes(stackFrame.filenameExtension)) {
+      return {
+        guide: (
+          <Docs
+            originFrameId={stackFrame.id}
```

Comment:
You need to add `lastUpdate={stackFrame.ts}` here for the reload to work

Reply:
Thank you :)